app/labor/reading_list.rb

```diff
@@ -28,6 +28,6 @@ def count
   end
 
   def reaction_criteria
-    { user_id: user.id, reactable_type: "Article", category: "readinglist" }
+    { user_id: user.id, reactable_type: "Article", category: "readinglist", status: "valid" }
```

Comment:
Rather than showing the incorrect counts for admins and doing only valid here why don't we just exclude the `archived` ones? So above 
```ruby
def ids_of_articles
  Reaction.where(reaction_criteria).where.not(status: "archived")...
end
```

Another option is rather than grabbing this count from the async info hash, get it from the Elasticsearch query.

Reply:
Ah, yeah, that's a much better solution. I'd rather do that than run a script on prod data. :sweat: